pkg/apis/messaging/v1beta1/kafka_channel_lifecycle.go

```diff
@@ -19,54 +19,43 @@ package v1beta1
 import (
 	"sync"
 
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
 	"knative.dev/pkg/apis"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 )
 
+// A default ConditionSet containing base configuration only for testing.
+// The consolidated and distributed KafkaChannel implementations require
+// differentiated ConditionSets in order to accurately reflect their varied
+// runtime architectures.  One of the channel specific "Register..." functions
+// in pkg/channel/<type>/apis/messaging/kafka_channel_lifecycle.go should be
+// called via an init() in the main() of associated components.
 var kc = apis.NewLivingConditionSet(
```

Comment:
If these are only for testing can we put them in a `*_test.go` file?

Reply:
👍